packages/main/src/ToolbarItem.ts

```diff
@@ -114,6 +124,14 @@ class ToolbarItem extends UI5Element {
 	get stableDomRef() {
 		return this.getAttribute("stable-dom-ref") || `${this._id}-stable-dom-ref`;
 	}
+
+	get class() {
+		return this._isOverflowed ? "ui5-tb-popover-item" : "ui5-tb-item" as string;
+	}
+
+	getFocusDomRef(): HTMLElement | undefined {
```

Comment:
This is needed because we've added a little "hack" in the toolbar, overriding the `_getRealDomRef` of each items https://github.com/SAP/ui5-webcomponents/blob/0ffbedecf962d319e5fb1a9fb495676dbe7a1c47/packages/main/src/Toolbar.ts#L558

If you check, even the `getDomRef` is not working.

Removing the hack (which is no longer needed) should fix both the `getDomRef` and `getFocusDomRef` methods, without any overrides.